cli/src/pcluster/aws/aws_resources.py

```diff
@@ -183,6 +183,36 @@ def gpu_count(self):
 
         return gpu_count
 
+    def gpu_manufacturers(self):
```

Comment:
`gpu_manufacturer` without the `s`. I do not expect an instance type have more than 1 gpu of different manufacturers.